.github/workflows/datahub-kafka-setup.yml

```diff
@@ -0,0 +1,83 @@
+name: datahub-kafka-setup docker
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - 'docker/kafka-setup/**'
+      - '.github/workflows/docker-kafka-setup.yml'
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+  pull_request:
+    branches:
+      - master
+    paths:
+      - 'docker/kafka-setup/**'
+      - '.github/workflows/docker-kafka-setup.yml'
+    paths_ignore:
+      - '**.md'
+      - '**.env'
```

Comment:
I don't think we want to ignore env files?

Reply:
Done! Also removed the docs ignore since it is not included in paths